src/components/Web3Status/index.tsx

```diff
@@ -128,7 +129,10 @@ const StyledConnectButton = styled.button`
 function Web3StatusInner() {
   const switchingChain = useAppSelector((state) => state.wallets.switchingChain)
   const ignoreWhileSwitchingChain = useCallback(() => !switchingChain, [switchingChain])
-  const { account, connector } = useLast(useWeb3React(), ignoreWhileSwitchingChain)
+  const connectionReady = useConnectionReady()
+  const activeWeb3 = useWeb3React()
+  const lastWeb3 = useLast(useWeb3React(), ignoreWhileSwitchingChain)
```

Comment:
i know you didn't add `useLast` but the name is a little confusing. i think `useMostRecent` would make more sense

Reply:
I'd support auditing `useLast` and `usePrevious` for both performance and naming. I think they each only have a couple of uses.